### PR TITLE
Django upgrade 4.2

### PR DESCRIPTION
**Pull request by salman2013**
**Description:**

- Add support for  Django 4.2

**Codemod used to upgrade this repo:**  https://github.com/adamchainz/django-upgrade

**Ticket:** https://github.com/edx/upgrades/issues/153

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @salman2013!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/.github/blob/master/CONTRIBUTING.md)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by editing the PR title.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

**Review by UsamaSadiq — APPROVED** [review]

File: organizations/admin.py [hunk 1]
```diff
@@ -27,6 +27,9 @@ def get_actions(self, request):
 
         return actions
 
+    @admin.action(
```
Comment (UsamaSadiq) [inline organizations/admin.py:30]: Please link the release notes for this change.

**Review by UsamaSadiq — CHANGES_REQUESTED** [review]

File: organizations/admin.py (hunk 1 above)
Comment (salman2013) [reply organizations/admin.py:30]: https://github.com/adamchainz/django-upgrade

**Comment by mphilbrick211** [comment]
Hi @salman2013 and @UsamaSadiq! Just following up on this.

**Pull request merged**